docs/content/storage-drivers/s3.md

```diff
@@ -44,7 +44,7 @@ Amazon S3 or S3 compatible services for object storage.
 
 `region`: The name of the aws region in which you would like to store objects (for example `us-east-1`). For a list of regions, see [Regions, Availability Zones, and Local Zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html).
 
-`regionendpoint`: (optional) Endpoint URL for S3 compatible APIs. This should not be provided when using Amazon S3.
+`regionendpoint`: (optional) Endpoint URL for S3 compatible APIs, from version 3+ required to use it with option forcepathstyle = true. Given the regionendpoint overrides the domain, forcing the path style is required, see [more about](https://github.com/distribution/distribution/issues/4528). **This option should not be provided when using Amazon S3.**
```

Comment:
```suggestion
`regionendpoint`: (optional) Endpoint URL for S3 compatible APIs, from version 3+ it's required to be used with `forcepathstyle: true`. Given the `regionendpoint` overrides the API host domain, forcing the path style is necessary, see [more about](https://github.com/distribution/distribution/issues/4528). **This option should not be provided when using Amazon S3.**
```

Reply:
Once you've either changed the text or accepted the commit suggestion I'm happy to approve this PR.

Reply:
done, its my first PR sorry